src/allmydata/frontends/sftpd.py

```diff
@@ -2011,5 +2011,5 @@ def getPrimes(self):
         f = SSHFactory()
         f.portal = p
 
-        s = strports.service(sftp_portstr, f)
+        s = strports.service(six.ensure_str(sftp_portstr), f)
```

Comment:
I assume that codecov says this line is uncovered because though it really is covered by the new integration tests and we have a bunch of stuff to try to collect coverage from the integration tests it is still somehow not integrated into the reports...

Reply:
It's definitely necessary for passing tests.